libbeat/outputs/elasticsearch/client_integration_test.go

```diff
@@ -55,6 +55,8 @@ func TestClientPublishEvent(t *testing.T) {
 }
 
 func TestClientPublishEventKerberosAware(t *testing.T) {
+	t.Skip("Flaky test: https://github.com/elastic/beats/issues/21295")
```

Comment:
20 fails of 20 builds it is not flaky is broken

Reply:
:slightly_smiling_face:

Reply:
We should skip it in any case by now.